openedx/core/djangoapps/discussions/serializers.py

```diff
@@ -354,6 +354,12 @@ def _update_course_configuration(
                     key not in LegacySettingsSerializer.Meta.fields_cohorts
                 )
             }
+        # toogle discussion tab is_hidden
```

Comment:
I think I would update this comment to explain a bit more, based on the comment you gave me. Perhaps:

```suggestion
        # Toggle discussion tab is_hidden. Before Palm, we would mark the discussion tab with the is_hidden property.
        # Redwood and later, we disable discussions entirely by toggling the discussion configuration enabled property.
        # This ensures pre-Palm courses import with discussions tab appropriately shown/hidden.
```